client/hostvolumemanager/host_volume_plugin.go

```diff
@@ -146,11 +152,11 @@ func (p *HostVolumePluginExternal) Version(ctx context.Context) (*version.Versio
 			"error", err)
 		return nil, fmt.Errorf("error getting version from plugin %q: %w", p.ID, err)
 	}
-	v, err := version.NewVersion(strings.TrimSpace(string(stdout)))
-	if err != nil {
-		return nil, fmt.Errorf("error with version from plugin: %w", err)
+	fprint := &PluginFingerprint{}
+	if err := json.Unmarshal(stdout, fprint); err != nil {
+		return nil, fmt.Errorf("error parson fingerprint output as json: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error parsing fingerprint output as json: %w", err)
```

Reply:
what, you don't like Jim Parsons of Big Bang Theory fame?

![parsons](https://people.com/thmb/V9G9-9IyPP8y-688dGuW0QYm0Dc=/750x0/filters:no_upscale():max_bytes(150000):strip_icc():focal(644x389:646x391)/jim-parsons-0a080c0832234195b66caf50436b56ab.jpg)

Reply:
it would be easy to search for that error message to debug. what if I do an ascii art of this image as a code comment to explain it?

(I am extra silly-headed today, don't mind me)